js/kraken.js

```diff
@@ -971,23 +971,7 @@ module.exports = class kraken extends Exchange {
         }
         return {
             'id': id,
-            'clientOrderId': clientOrderId,
```

Comment:
I think clientOrderId is also ok to keep defined. It doesn't really change over time.

Reply:
yep